Fix npm audit fail issue 4803

g-k opened the pull request:
Assuming no new insecure deps, this should fix #4803 (for CI at least).

6a68 commented:
> Sorry about all the spam in #screenshots on irc.

I'll take free help + irc spam any time you have spare cycles ^_^

chenba commented:
The npm version required is a problem locally until #4804 is fixed.  (Or we could document that as a requirement for the project).

g-k commented:
OK made some changes:

* added mac support in https://github.com/mozilla-services/audit-filter/issues/3
* stripped the binaries; makes them almost 10x smaller
* moved the shell script into git (in `example/bin`) https://github.com/mozilla-services/audit-filter/#install and pulled in the rustup arch detection

Haven't tested this PR on OSX (have tested audit-filer), but try running: `CI=1 npm run-script lint:deps`
